tests/test_due_diligence_verification.py

```
@@ -0,0 +1,469 @@
+                    for field in required_fields:
+                        assert field in doc, f"Missing field '{field}' in {category} document: {doc.get('name', 'unnamed')}"
+                    
+                    # Check field types
+                    assert isinstance(doc["name"], str)
+                    assert isinstance(doc["status"], bool)
+                    assert isinstance(doc["visibility"], list)
+        
+        # Separate pass for equipment-specific validation
+        if "equipment" in internal_data and "documents" in internal_data["equipment"]:
+            for doc in internal_data["equipment"]["documents"]:
+                assert "value" in doc, f"Missing 'value' field in equipment item: {doc['name']}"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)